optimize.rb

```diff
@@ -79,7 +79,7 @@ def line(name, old_size, new_size)
   # Escape %, \n, and \r.
   # Ref: https://github.community/t/set-output-truncates-multiline-strings/16852/3
   summary.gsub!(/[%\n\r]/, "%" => "%25", "\n" => "%0A", "\r" => "%0D")
-  puts "::set-output name=summary::#{summary}"
+  puts "echo ':name=summary::#{summary}' >> $GITHUB_OUTPUT"
```

Comment:
You are correct of course. Lol, that's what I get for copying from GitHub 😅.

Took another attempt at it, it's a little different from your suggestion above. Why do we need to redirect stdout, is it only because we're using `puts` there?